src/components/workspace-utils.js

```diff
@@ -28,17 +28,15 @@ export const useWorkspaces = () => {
 }
 
 export const withWorkspaces = WrappedComponent => {
-  const Wrapper = props => {
+  return Utils.withDisplayName('withWorkspaces()', props => {
```

Comment:
The recent devtools seems to strip off the parens when displaying these names. Since we're updating this code, it might be nice to remove the parens here to avoid confusion.